Correct typo

jacobherrington opened the pull request:
"in about its first four years" doesn't quite make sense. This change
should be a bit more clear!

Additionally, I felt it the end of this sentence could be made more
clear as well.

fxn commented:
Hmmm, unsure about these edits.

* The first one is missing that those four years were the first four years.
* Git does not have commit "authors", it has commit "author". For example, from `git commit -h`: `--author <author>     override author for commit`.
* To me, "readily available" sounds too weak. Subversion does not have the concept of "author", it is missing in their model. It is a commit attribute that just does not exist.

jacobherrington commented:
@fxn I updated the text to reflect your comments and rebased.

Thanks for the feedback, let me know if this still misses the mark. :slightly_smiling_face:
